brainstormforce/ultimate-addons-for-gutenberg
# Fix: CTA - Icon default color, Alignment in backward and heading/description spacing.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/ed1d9717-0ff7-4106-acec-7c60f1fd03a9?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- For Backword
  - Create a CTA in master with button position > stack and alignment center.
  - Switch to next-release and see the issue mentioned in the card is resproduce.
  - Switch to the above branch to fix the issue.
- For New User 
  - Drag and drop CTA and add an icon to the button and check the frontend.
  - Default Icon color should be the same as the text.
  - Check Heading/Description is working in editor as well as on frontend.
  - Check Preset design.

### Checklist:
